sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/IntervalUtils.scala

```diff
@@ -386,6 +386,7 @@ object IntervalUtils {
   }
 
   def divide(interval: CalendarInterval, num: Double): CalendarInterval = {
```

Comment:
we should move `add`, `subtract`, `negate` to `IntervalUtils` as well. Maybe we need to change the overflow behavior later on.

@yaooqinn can you do this later?

Reply:
Yes, I am willing to do this, thanks.